vm/actor/src/lib.rs

```diff
@@ -50,3 +49,33 @@ pub trait Actor {
     /// Actor public key, if it exists
     fn public_key(&self) -> Vec<u8>;
 }
+
+impl ser::Serialize for ActorState {
+    fn serialize<S>(&self, s: S) -> Result<S::Ok, S::Error>
+    where
+        S: ser::Serializer,
+    {
+        (
+            self.code_id.clone(),
+            self.state.clone(),
+            self.sequence,
+            self.balance.clone(),
+        )
+            .serialize(s)
+    }
+}
+
+impl<'de> de::Deserialize<'de> for ActorState {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: de::Deserializer<'de>,
+    {
+        let (code_id, state, sequence, balance) = Deserialize::deserialize(deserializer)?;
+        Ok(Self {
+            code_id,
+            state,
+            sequence,
+            balance,
+        })
+    }
+}
```

Comment:
I don't see why this is needed (extension of serializing CodeID) I removed from this PR with no issue